packages/mgt-components/src/components/mgt-person-card/mgt-person-card.ts

```diff
@@ -500,7 +504,7 @@ export class MgtPersonCard extends MgtTemplatedComponent {
             <div class="nav">
               <div class="nav__back" tabindex="0" @keydown=${(e: KeyboardEvent) => {
```

Comment:
Can we make this a real button? or at the very least add the role=button attribute and an aria-label?

Reply:
Changing this to a button